crates/chain/src/changeset.rs

```diff
@@ -1,5 +1,4 @@
 /// A changeset containing [`crate`] structures typically persisted together.
-#[cfg(feature = "miniscript")]
```

Comment:
If it's not necessary part of the PR revert it.

The correct way to clean this up is to remove it. If we're not going to remove it yet at least call the file `wallet_changeset.rs` and the type `WalletChangeSet` with a big notice at the top saying "This is going to be moved soon". It's confusing to all it `changeset.rs` because people will want to know how to create their own changeset but the trait for that is actually called `Append`.

Reply:
Removed.